docs/UserGuide.md

```diff
@@ -129,7 +129,8 @@ Format birthday as `ddMMyyyy`.
 </div>
 
 Notes:
-* Phone number must be unique.
+* Phone number must be unique and must contain only numbers.
+* Emails must contain @. (Other stricter checks are not included as there are some emails that violate the convention).
```

Comment:
Would something along the lines of "Valid emails that violate convention are accepted as well"